### ISSUE TITLE
Looking for a Rich Text Editor on the WET

### ISSUE THREAD

**Issue by Jibombardier** [wet-boew/wet-boew#8746]
I would like to know if there will be a rich text editor in the WET soon? or if we have to use a plugin. And if so, is there a recommended accessible plugin?
thank you

**Comment by crazybat** [comment]
[CKEditor](https://ckeditor.com) is currently citing WCAG Level AA compliant. It's also fairly active, in regards to [current issues on Github](https://github.com/ckeditor/ckeditor5/issues)

**Comment by Jibombardier** [comment]
Thanks I will look into it.

**Comment by juleskuehn** [comment]
I agree that there should be a default rich text editor in WET, available by adding a class `.wb-rte` to a textarea. And agree with crazybat about CKEditor, but the versions differ:

CKEditor 4 is the best for accessibility. You may need to install additional plugins, for instance the language plugin.

I can't recommend CKEditor 5 at this time. It has a number of improvements (cleaner UI, better for mobile etc.) and is generally accessible - but Image and Table plugins both have accessibility issues. As well, language tagging of the content is not supported. CKEditor 5 also doesn't work with IE11 as it is written with ES6 features.

**Comment by Jibombardier** [comment]
Thank you for this information. I found out though that CKEditor is not free which means we would need to purchase this plugin instead of a free plugin to install on GIT... any other that would be free that you can recommend?

**Comment by thekodester** [comment]
There site can be confusing, but CKEditor 4 is free.

See https://ckeditor.com/ckeditor-4/download/.

**Comment by crazybat** [comment]
Indeed, I thought CKEditor was free, as per the link by @thekodester :\

**Comment by Jibombardier** [comment]
oh great! I saw an enterprise price yesterday....somewhere. wonderful thank you.

**Comment by thekodester** [comment]
@Jibombardier Yeah the developers do have a pricing model, but that is only for direct support.

**Comment by Jibombardier** [comment]
great thanks will proceed with the CKEditor 4 as recommended. thanks a lot :-)

**Comment by thekodester** [comment]
OR commercial use (non-open source projects or projects that produce money using there software).

**Comment by Jibombardier** [comment]
oh ok.  any of you tried this? [Quilljs](https://quilljs.com/) * WYSIWYG / Rich text editor that are API based
I found it as a reference on : https://wet-boew.github.io/wet-boew-documentation/research/2018-6-wb5-exploration-1.html   don't know if it is accessible though...and it doesn't support IE (although IE is coming shortly to an official end...)

**Comment by thekodester** [comment]
As @juleskuehn said not accessible. Also there GitHub page shows a number of accessibility issues that where brought up, but not addressed. Also there GitHub shows a lack of active development.

**Comment by Jibombardier** [comment]
good to know. thanks again
